src/Makefile.am

```diff
@@ -1 +1,3 @@
-dist_bin_SCRIPTS = eos-google-chrome-app
+dist_bin_SCRIPTS =		\
+	eos-google-chrome-app	\
+	eos-google-chrome-dummy-cmd
```

Comment:
I wanted to make sure we had a separate script so it was clear but I like your proposal too. I'll change that.